From 9ec2efa6ba4bfaed7805867077d3a67daa022331 Mon Sep 17 00:00:00 2001
From: Daniel Engberg <daniel.engberg.lists@pyret.net>
Date: Mon, 27 Jun 2022 01:58:45 +0200
Subject: [PATCH] cmake: Add aarch64 as valid match for 64-bit (#5)

Allow aarch64 aka arm64 to use 64-bit fixed point math
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -78,7 +78,7 @@ elseif(CMAKE_SYSTEM_PROCESSOR MATCHES ".
 elseif(CMAKE_SYSTEM_PROCESSOR MATCHES ".(ppc|PPC).*")
   message(STATUS "Using PowerPC fixed point math")
   option(FPM_PPC "PowerPC fixed point math" ON)
-elseif(CMAKE_SYSTEM_PROCESSOR MATCHES ".*(x86_64|x86-64|x64|AMD64|amd64).*")
+elseif(CMAKE_SYSTEM_PROCESSOR MATCHES ".*(x86_64|x86-64|x64|AMD64|amd64|aarch64).*")
   message(STATUS "Using 64 bit fixed point math")
   option(FPM_64BIT "64 bit fixed point math" ON)
 else()
